src/modules/controllers/task-controllers.js

```diff
@@ -0,0 +1,107 @@
+const Task = require('../../modul/task.js');
+
+const getAllTasks = async (req, res) => {
+  try {
+    const allTasks = await Task.find();
+    res.status(200).send(allTasks);
+  } catch (error) {
+    res.status(400).send('Task retrieval error');
+  }
+};
+
+const createNewTask = async (req, res) => {
+  try {
+    const text = req.body.text;
+    const cost = req.body.cost;
+    const date = req.body.date;
```

Comment:
You can replace it using destructurisation like:
const { text, date, cost } = req.body